scripts/release.js

```diff
@@ -123,8 +123,15 @@ if (process.platform == "linux") {
       CFBundleVersion: `${package.version}`,
       CFBundlePackageType: "APPL",
       CFBundleSignature: "????",
-      CFBundleExecutable: "Oni2",
+      CFBundleExecutable: "Oni2_editor",
```

Comment:
This looks good, but we'll need to port over this CLI logic:
https://github.com/onivim/oni2/blob/8c600e062746eb647dd8270cbc770de2365f51dc/src/bin_launcher/Oni2.re#L97

Reply:
I noted that in the original PR comment 👍. I was looking at `Cli.re`, but it seems to me that we dont pass in `Sys.argv` like we do in `Oni2`. Not sure what the best solution for that is, unfortunately.

Reply:
Ah, ya, we need to use a different call - `Arg.argv_parse` - that lets us pass in a list of arguments (`Argv.parse` is like `Arg.argv_parse` with `Sys.argv` passed in). 

I made the change here: https://github.com/onivim/oni2/pull/1795 (and wanted to add some test coverage on our CLI anyway, so this was a good opportunity)